pkg/kudoctl/cmd/package_list_plans.go

```diff
@@ -97,7 +97,10 @@ func displayPlanTable(pf *packages.Files, withTasks bool, out io.Writer) {
 }
 
 func sortedPlanNames(pf *packages.Files) []string {
-	planNames := funk.Keys(pf.Operator.Plans).([]string)
+	planNames, ok := funk.Keys(pf.Operator.Plans).([]string)
+	if !ok {
+		panic("funk.Keys returned unexpected type")
```

Comment:
We don't `panic`: for the most part we `os.Exit(1)` with some log message before. It gives us the chance to introduce different exit codes more smoothly later. Otherwise, LGTM